src/LibDeclarations/browser.d.ts

```diff
@@ -36,6 +36,13 @@ interface CanvasRenderingContext2D {
     msImageSmoothingEnabled: boolean;
 }
 
+// Babylon Extension to enable Native compatibility

```

Comment:
nit: these properties aren't just for native

Reply:
Updated comment to reflect event compatibility, rather than just native compatibility